R/kernel.r

```diff
@@ -129,6 +129,35 @@ handle_shell = function() {
         print(c('Got unhandled msg_type:', msg$header$msg_type)))
 },
 
+handle_abort = function(){
```

Comment:
Can we call this something like `abort_shell_msg`? `handle_x` usually means that there is an x arriving that we need to deal with.